grails-app/conf/templates/configuration/products.csv

```diff
@@ -1,10 +1,10 @@
 Id,ProductCode,ProductType,Name,ProductFamily,Category,GLAccount,Description,UnitOfMeasure,Tags,UnitCost,LotAndExpiryControl,ColdChain,ControlledSubstance,HazardousMaterial,Reconditioned,Manufacturer,BrandName,ManufacturerCode,ManufacturerName,Vendor,VendorCode,VendorName,UPC,NDC,Created,Updated
 ,QX039,Default,Morphine 10mg immediate release tablet,,Pain,,,Each,Tertiary Care,0.25,TRUE,,TRUE,,,,,,,,,,,,,
 ,TS894,Default,"Morphine injection, 10mg/mL in 1mL ampoule",,Pain,,,Each,Tertiary Care,1.25,TRUE,,TRUE,,,,,,,,,,,,,
-,CR388,Default,Lamivudine 150mg + nevirapine 200mg + zidovudine 300mg tablet,,ARVs,,,Each,Primary Care,0.29,TRUE,,,,,,,,,,,,,,,
-,TZ835,Default,Lamivudine 150mg + zidovudine 300mg tablet,,ARVs,,,Each,Primary Care,0.29,TRUE,,,,,,,,,,,,,,,
-,CM871,Default,Lamivudine 150mg tablet,,ARVs,,,Each,Primary Care,0.29,TRUE,,,,,,,,,,,,,,,
-,MC017,Default,Lamivudine 30mg + nevirapine 50mg + zidovudine 60mg tablet,,ARVs,,,Each,Primary Care,0.29,TRUE,,,,,,,,,,,,,,,
+,CR388,Default,Lamivudine 150mg + nevirapine 200mg + zidovudine 300mg tablet,,ARVS,,,Each,Primary Care,0.29,TRUE,,,,,,,,,,,,,,,
+,TZ835,Default,Lamivudine 150mg + zidovudine 300mg tablet,,ARVS,,,Each,Primary Care,0.29,TRUE,,,,,,,,,,,,,,,
+,CM871,Default,Lamivudine 150mg tablet,,ARVS,,,Each,Primary Care,0.29,TRUE,,,,,,,,,,,,,,,
+,MC017,Default,Lamivudine 30mg + nevirapine 50mg + zidovudine 60mg tablet,,ARVS,,,Each,Primary Care,0.29,TRUE,,,,,,,,,,,,,,,
```

Comment:
Take a look at `default_OB_categories.csv` (and `changelog-insert-data-core.groovy` but I don't think that's used). The categories are created with ARVS but this file tried to set ARVs.

The code change in ProductService makes this csv change unnecessary (because it doesn't case-insensitive matching), but I figured it'd be best to keep the data in sync.